src/modules/i18n/containers/I18nProvider.js

```diff
@@ -81,7 +81,7 @@ export class I18nProvider extends React.Component<PropsType, StateType> {
     )
   }
 
-  setLanguage (language: ?string) {
+  async setLanguage (language: ?string) {
```

Comment:
Right now `setLanguage` is not really async since you don't `await` the promise, is it?

Reply:
It would probably be better code to just  use `async/await` in this case here, since the then callback doesn't even use the value returned by `this.i18n.changeLanguage`